### ISSUE TITLE
Leave realm action doesn't work from My Account page

### ISSUE THREAD

**Issue by bschlaman** [google/exposure-notifications-verification-server#1987]
### TL;DR
When attempting to leave a realm from the account page (for example, if you are a member of several realms and want to remove yourself from several at once), the "Leave realm" button doesn't seem to work.

- On some instances, an error message is shown
- On other instances, "Unauthorized" message is displayed.  This occurs for both the realm you have selected, as well as other realms.

I think the behavior you get has to do with if you have a realm selected for your session, although I'm not 100% sure.

**Expected behavior**
Leave realm button removed you from the realm.

**Observed behavior**
Either an error message is shown or an Unauthorized screen appears.

### Reproduction

1. Log into console as sys admin
2. Navigate to "My account"
3. Try to leave a realm from this page

**Environment**
- OS: Windows
- Browser: Chrome
- Version: Version 89.0.4389.90 (Official Build) (64-bit)

**Additional information**

- There are instances where I (a sys admin) have been explicitly added to realms by someone else.  Not sure if that would have any impact here.
- I've observed the behavior in more than one mvs env

**Comment by sethvargo** [comment]
Hey @bschlaman 

This button is actually removed at HEAD (as of yesterday) because of its undefined behavior. You only saw it as a sysadmin, so it's not removed functionality for individual users. You'll still see the list of realms, but you need to join that realm to leave it. You can also leave realms from the system admin UI.
